commercial/app/views/hosted/guardianHostedVideo.scala.html

```diff
@@ -5,6 +5,64 @@
 @import views.html.hosted.guardianHostedCta
 
 @main(page, Some("commercial")) { } {
+    <!--[if (gt IE 9)|(IEMobile)]><!-->
+    <style>
+    .hosted-tone {
+        color: @{page.campaign.brandColour};
+    }
+
+    .hosted-tone-bg {
+        background-color: @{page.campaign.brandColour};
+    }
+
+    .hosted-tone-btn,
+    .hosted-tone-btn:focus,
+    .hosted-tone-btn:hover {
+        background-color: @{page.campaign.brandColour};
+        border-color: @{page.campaign.brandColour};
+    }
+
+    .hosted-page .hosted-next-autoplay .hosted-next-autoplay__poster-timer,
+    .hosted-page .hosted-next-autoplay .hosted-next-autoplay__tile {
+        background-color: @{page.campaign.brandColour};
+    }
+
+    .hosted-page .hosted-next-autoplay .hosted-next-autoplay__video,
+    .hosted-page .hosted-next-autoplay .hosted-next-autoplay__next-in-series {
+        border-top: 1px solid @{page.campaign.brandColour};
+    }
+
+    .hosted-page .hosted__next-video .hosted__next-video--tile {
+        border-top-color: @{page.campaign.brandColour};
+        background: rgba(@{page.campaign.brandColour}, 0.5); /* TODO transform hex into rgb in js */
+    }
+
+    .hosted-page .hosted__link {
+        color: @{page.campaign.brandColour};
+    }
+
+    .hosted-page ~ .survey-overlay-simple .survey-text__header {
+        background-color: @{page.campaign.brandColour};
+    }
+
+    @if(page.campaign.brightFont) {
+        .hosted-tone-btn,
+        .hostedbadge.hosted-tone-bg .hostedbadge__info,
+        .hosted-page ~ .survey-overlay-simple .survey-text__header {
+            color: #ffffff;
```

Comment:
I am not sure if I understand you. There will be two font colours schemes: white and black. User will only specify which scheme will look better for the campaign/brand colours.